src/z3c/testing/layer.py

```diff
@@ -43,7 +40,7 @@ class BufferedDatabaseTestLayer(object):
     __bases__ = ()
     path = None
 
-    def __init__(self, config_file=None, module=__module__,
+    def __init__(self, config_file=None, module=__module__,  # noqa
```

Comment:
Is there a pyflakes bug about this false positive warning?  `__module__` is always defined in class scope:

```
>>> __module__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name '__module__' is not defined
>>> class Foo:
...   print(__module__)
... 
__main__
```